cfspopcon/formulas/separatrix_operational_space/LH_transition_power.py

```diff
@@ -0,0 +1,61 @@
+"""Routines to calculate the separatrix power required to reach the LH transition."""
+import matplotlib.pyplot as plt
+import xarray as xr
+from scipy.interpolate import RectBivariateSpline  # type:ignore[import-untyped]
+
+from cfspopcon.unit_handling import Unitfull, convert_units, magnitude, ureg
+
+
+def extract_LH_contour_points(LH_transition_condition: xr.DataArray) -> tuple[xr.DataArray, xr.DataArray]:
+    """Extract the density and temperature along the LH transition line."""
+    # Hacky way of getting contour points
+    _, ax = plt.subplots()
+    contour_set = LH_transition_condition.T.plot.contour(levels=[1.0], ax=ax)
```

Comment:
https://github.com/cfs-energy/cfspopcon/pull/12#discussion_r1465384278

Use skiimage instead of matplotlib for contours.

Reply:
Using contourpy. https://contourpy.readthedocs.io/en/v1.2.1/quickstart.html